collector/lib/CollectorConfig.cpp

```diff
@@ -160,6 +163,48 @@ CollectorConfig::CollectorConfig(CollectorArgs* args) {
   HandleAfterglowEnvVars();
 
   host_config_ = ProcessHostHeuristics(*this);
+
+  const char* envvar;
+
+  connection_stats_quantiles_ = {0.50, 0.90, 0.95};
+
+  if ((envvar = std::getenv("ROX_COLLECTOR_CONNECTION_STATS_QUANTILES")) != NULL) {
+    connection_stats_quantiles_.clear();
+    std::stringstream quantiles(envvar);
+    while (quantiles.good()) {
+      std::string quantile;
+      std::getline(quantiles, quantile, ',');
+      try {
+        double v = std::stod(quantile);
+        connection_stats_quantiles_.push_back(v);
+        CLOG(INFO) << "Connection statistics quantile: " << v;
+      } catch (...) {
+        CLOG(ERROR) << "Invalid quantile value: '" << quantile << "'";
+      }
+    }
+  }
+
+  connection_stats_error_ = 0.01;
+
+  if ((envvar = std::getenv("ROX_COLLECTOR_CONNECTION_STATS_ERROR")) != NULL) {
+    try {
+      connection_stats_error_ = std::stod(envvar);
+      CLOG(INFO) << "Connection statistics error value: " << connection_stats_error_;
+    } catch (...) {
+      CLOG(ERROR) << "Invalid quantile error value: '" << envvar << "'";
+    }
+  }
```

Comment:
This could also be moved to its own function elsewhere to reduce clutter.

Reply:
Moved all the connection-stats specific to a dedicated method, in the same spirit as Afterglow parsing.

Does this look ok this way ?